pkg/operator/ceph/object/admin.go

```diff
@@ -110,8 +111,50 @@ func runAdminCommand(c *Context, expectJSON bool, args ...string) (string, error
 			fmt.Sprintf("--rgw-zone=%s", c.Zone),
 		}
 
-		return RunAdminCommandNoMultisite(c, expectJSON, append(args, options...)...)
+		args = append(args, options...)
 	}
 
-	return RunAdminCommandNoMultisite(c, expectJSON, args...)
+	// work around https://github.com/rook/rook/issues/7573 (FIFO file I/O issue)
```

Comment:
I'd suggest we include issue numbers just in the PR or commit description, and the code would only describe what's being fixed

Reply:
I left the issue number without the full URL just in case someone comes back and wants more information without having to look up the commit, then PR, then issue.